modules/test/playwright/tests/frontend-data-set-views-web/itemActions.spec.ts

```diff
@@ -9,68 +9,941 @@ import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+			await viewsPage.createDataSetView({name: DATASET_VIEW_NAME});
+		});
+
+		await test.step('Go to Actions tab', async () => {
+			await actionsPage.goto({
+				dataSetName: DATASET_NAME,
+				dataSetViewName: DATASET_VIEW_NAME,
+			});
+		});
+
+		await test.step('Create an item action', async () => {
+			await actionsPage.createItemAction({
+				icon: 'link',
+				name: LINK_ITEM_ACTION_NAME,
+				type: 'link',
+				url: 'https://www.liferay.com/',
+			});
+		});
 
-	await test.step('Go to Actions tab', async () => {
-		await actionsPage.goto({
-			dataSetName: DATASET_NAME,
-			dataSetViewName: DATASET_VIEW_NAME,
+		await test.step('Check that the item action is in the list', async () => {
+			await expect(
+				page
+					.getByRole('cell', {
+						exact: true,
+						name: LINK_ITEM_ACTION_NAME,
+					})
+					.locator('span')
+					.first()
+			).toBeVisible();
+		});
+
+		await test.step('Delete Data Set', async () => {
+			await dataSetsPage.deleteDataSet(DATASET_NAME);
 		});
 	});
 
-	await test.step('Create an item action', async () => {
-		await actionsPage.createItemAction({
-			icon: 'link',
-			name: LINK_ITEM_ACTION_NAME,
-			type: 'link',
-			url: 'https://www.liferay.com/',
+	test('Link Item Action is shown in fragment', async ({
+		actionsPage,
+		dataSetsPage,
+		fdsFragmentPage,
+		fieldsPage,
+		page,
+		viewsPage,
+	}) => {
+		const DATASET_NAME = 'Item Actions DS';
+		const DATASET_VIEW_NAME = 'Item Actions DS View';
+		const LINK_ITEM_ACTION_CONFIRMATION_MESSAGE =
+			'Do you want to navigate to http://www.liferay.com?';
+		const LINK_ITEM_ACTION_NAME = 'Link item action';
+		const PAGE_NAME = 'Test page';
+		const SITE_NAME = 'FDSFragmentSite';
+
+		await test.step('Create Data Set', async () => {
+			await dataSetsPage.goto();
+			await dataSetsPage.createDataSet({name: DATASET_NAME});
+		});
+
+		await test.step('Create Data Set View', async () => {
+			await viewsPage.goto(DATASET_NAME);
+			await viewsPage.createDataSetView({name: DATASET_VIEW_NAME});
+		});
+
+		await test.step('Open modal to add fields', async () => {
+			await fieldsPage.goto({
+				dataSetName: DATASET_NAME,
+				dataSetViewName: DATASET_VIEW_NAME,
+			});
+			await fieldsPage.openAddFieldsModal();
+		});
+
+		await test.step('Select fields in treeview', async () => {
+			await fieldsPage.addRootField('id');
+			await fieldsPage.addRootField('name');
+		});
+
+		await test.step('Save changes', async () => {
+			await fieldsPage.saveAddFieldsModal();
+		});
+
+		await test.step('Go to Actions tab', async () => {
+			await actionsPage.goto({
+				dataSetName: DATASET_NAME,
+				dataSetViewName: DATASET_VIEW_NAME,
+			});
+		});
+
+		await test.step('Create a link item action', async () => {
+			await actionsPage.createItemAction({
+				confirmationMessage: LINK_ITEM_ACTION_CONFIRMATION_MESSAGE,
+				icon: 'link',
+				name: LINK_ITEM_ACTION_NAME,
+				type: 'link',
+				url: 'https://www.liferay.com/',
+			});
+		});
+
+		const siteInfo =
```

Comment:
To me, it makes sense to create a helper that handles lines 145 -> 214: create a site, create a page, drop the fragment, select the view, publish ... and avoid repetition for each test that uses the fragment.
The same could apply for field selection. What do you think?